src/app/home/home.component.html

```diff
@@ -53,10 +53,16 @@ <h1>New Space Dashboard!</h1>
           <div class="card-pf f8-card-lg">
             <div class="card-pf-heading f8-card-heading">
               <div class="card-pf-heading-details f8-card-heading-details">
-                <a class="btn btn-link f8-card-heading-btn-link" (click)="openForgeWizard(createSpace)">
-                  <!-- <a class="btn btn-link f8-card-heading-btn-link" (click)="showAddSpaceOverlay()"> -->
```

Comment:
We should probably remove the commented out code

Reply:
Handled.

Reply:
To be clear, I was speaking about cleaning up the commented-out links, buttons, etc. in all changed files.

Reply:
Dan, sorry about that. Will be fixing it.